src/main/webapp/i18n/de/complaint.json

```diff
@@ -27,6 +27,8 @@
             "rejected": "Abgelehnt",
             "rejectedLong": "Beschwerde wurde abgelehnt",
             "tooManyComplaints": "Maximale Anzahl an Beschwerden erreicht. Du kannst nicht mehr als {{ maxComplaintNumber }} offene oder abgelehnte Beschwerden gleichzeitig haben.",
+            "exceededComplaintTextLimit": "Maximale Anzahl an Zeichen erreicht. Du kannst nicht mehr als {{ maxComplaintTextLimit }} Zeichen verwenden.",
+            "exceededComplaintResponseTextLimit": "Maximale Anzahl an Zeichen erreicht. Du kannst nicht mehr als {{ maxComplaintRespondTextLimit }} Zeichen verwenden.",
```

Comment:
These two translations are the same. Either make them distinct or reduce them to a single translation.

Reply:
Good point. I have adjusted it accordingly. I think separating these two translations makes it more adaptable in the future.